01-programming-core-mql4/03-operations-on-variables.mq4

```diff
@@ -21,6 +21,15 @@ void OnStart()
    z = x + y;
    Alert(z);               //should give 23 as output
    
+   //Substraction Operation
+   
+   int x = 15;
+   int y = 8;
+   int z;
+   
+   z = x - y;
+   Alert(z);               //should give 23 as output
```

Comment:
Change the comment in subtraction. 15 - 8 shouldn't give 23 as output.

Reply:
yes , it should be 7